contracts/extension/GovernorVotesSuperQuorumFractionUpgradeable.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MIT
+// Original design by OpenZeppelin
+// Modified by Dennison Bertram and Arthur Abeilice  @ Tally.xyz
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/governance/extensions/GovernorVotes.sol";
```

Comment:
I think there is a `contracts-upgradable` version of all the OZ Governance contracts as well. We should be using all of those.